BPOR/BPOR.Domain/Migrations/20241119185555_CRNCC-2394-adding-letter-campaign-functionality.cs

```diff
@@ -0,0 +1,203 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+#pragma warning disable CA1814 // Prefer jagged arrays over multidimensional
+
+namespace Dynamo.Stream.Handler.Migrations
+{
+    /// <inheritdoc />
+    public partial class CRNCC2394addinglettercampaignfunctionality : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_CampaignParticipants_Campaigns_EmailCampaignId",
```

Comment:
shouldn't be dropping and recreating these tables. migration should be renaming instead.
noticed by Thomas on another PR